indexer/src/main/java/uk/ac/ebi/uniprot/indexer/taxonomy/readers/TaxonomyNodeReader.java

```diff
@@ -0,0 +1,43 @@
+package uk.ac.ebi.uniprot.indexer.taxonomy.readers;
+
+import org.springframework.jdbc.core.RowMapper;
+import uk.ac.ebi.uniprot.indexer.taxonomy.TaxonomyDocument;
+import uk.ac.ebi.uniprot.indexer.taxonomy.steps.TaxonomyNodeStep;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+/**
+ * This class is mapping {@link ResultSet} returned by SQL executed in
+ *  {@link TaxonomyNodeStep}
+ *  to {@link TaxonomyDocument} object that will be used to save Taxonomy Nodes
+ *
+ *  @author lgonzales
+ */
+public class TaxonomyNodeReader implements RowMapper<TaxonomyDocument> {
+
+    @Override
+    public TaxonomyDocument mapRow(ResultSet resultSet, int i) throws SQLException {
+        TaxonomyDocument.TaxonomyDocumentBuilder builder = TaxonomyDocument.builder();
+        builder.id(""+resultSet.getLong("TAX_ID"));
```

Comment:
Use String.value method.
Make the string constant instead of hard coded string.